integration-tests/maven/src/test/resources/projects/multimodule-revision-prop/html/src/main/resources/META-INF/resources/a.html

```diff
@@ -0,0 +1 @@
+sadfasdfasdfas
```

Comment:
added by mistake or ?

Reply:
This is just a copy of the multimodule project that already existed in the TS.